06-Tutorial-CSR.Rmd

```diff
@@ -848,12 +848,19 @@ fmean_narm <- function(x){mean(x,na.rm=T)}
 
 age_region_test <- aggregate(cv_tot$age, by=list(cv_tot$region), FUN=fmean_narm)
 head(age_region_test)
-
 ```
 
-### Tidy  {#agtidy}
+### Tidy
+```{r, results='hold'}
+#step 1. convert the dataset to an aggregate/grouped version using the "group_by" function from the Dplyr package (part of the tidyverse), which will enable you to perform aggregate-level, or "by group," operations.
+age_region <- group_by(cv_tot, region)
 
-** TO DO ** 
+#step 2. use "summarise" (also from the dplyr package) to create a new data frame containing the mean age per region. Note: summarise() automatically creates a new data frame.
+age_region <- summarise(by_region, mean = mean(age))
```

Comment:
How do you deal with possible missing values in `age`.
Please note the typo. It should be:

`age_region <- summarise(age_region, mean = mean(age))`